### PR TITLE
Add Subscribe to an API document page

### PR DESCRIPTION
**Pull request by tmkasun**
## Purpose
 This includes the document updates for  `Subscribe to an API` page

### CODE REVIEW

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md [hunk 1]
```diff
@@ -1,195 +1,201 @@
...
 
-4.  Enter the name as TestApp and select the per token quota as 50PerMin for the application and click **Add** .
-    ![]({{base_path}}/assets/attachments/103327837/103327821.png)
-5.  Click **APIs** and click the PhoneVerification API to view the API's subscription options.
-6.  Select the application that you just created, a tier, and click **Subscribe** .
-    ![]({{base_path}}/assets/attachments/103327837/103327820.png)
-7.  Click the **View Subscriptions** button when prompted.
-    The **Subscriptions** tab opens.
+!!! tip
+    In a [multi-tenanted](https://docs.wso2.com/display/AM210/Configuring+Multiple+Tenants) WSO2 API Manager setup, you can access any tenant's store using the URL `http://<hostname>:<port>/store?tenant=<tenant_name>` .
```
Comment (AmaliMatharaarachchi) [inline en/docs/Learn/Tutorials/subscribe-to-an-api.md]: Shall we change store to developer portal?

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md [hunk 2]
```diff
@@ -1,195 +1,201 @@
...
 
-        In the above example, the placeholders mentioned in the cURL command format are replaced as follows:
 
-        -   **&lt;access token&gt;** : Give the test token generated in step 8. Click **Applications** , click on the respective application, which in this case is TestApp, click **Production Key** , and click **copy button** to copy the access token.
+In the above example, the placeholders mentioned in the cURL command format are replaced as follows:
 
-                        !!! warning
-            Make sure you have updated the flash plugin in your web browser in order to get the **copy button** working.
+-   **&lt;access token&gt;** : Give the test token generated in step 8.
+-   **&lt;API URL&gt;** : Click the respective API, in this case, PhoneVerification - 1.0.0. When the API's **Overview** tab appears in the API Store, copy the production URL and append the required query params to it.
```
Comment (AmaliMatharaarachchi) [inline en/docs/Learn/Tutorials/subscribe-to-an-api.md]: Here as well.

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md [hunk 3]
```diff
@@ -1,195 +1,201 @@
...
-        !!! tip
+!!! tip
     Access Tokens with specific Scopes
 
     Access tokens can be generated for specific scopes. A scope acts as a limiting factor on what API resources can be accessed using a token. To generate an access token corresponding to a scope, use the drop down menu under **Scopes** and select the required scope parameter.
 
-        !!! note
-    If you are using WSO2 Identity Server 5.6.0 as the Key Manager for your API Manager deployment, generating keys will result in the creation of a [Service Provider](https://docs.wso2.com/display/IS560/Configuring+a+Service+Provider) in the Identity Server.
+!!! note
+    If you are using WSO2 Identity Server 5.9.0 as the Key Manager for your API Manager deployment, Clicking __NEXT__ will result in the creation of a [Service Provider](https://docs.wso2.com/display/IS590/Configuring+a+Service+Provider) in the Identity Server.
```
Comment (AmaliMatharaarachchi) [inline en/docs/Learn/Tutorials/subscribe-to-an-api.md]: Is this IS URL correct? However, there is a https://is.docs.wso2.com/en/5.9.0/setup/configuring-a-sp-and-idp-using-configuration-files/

Comment (tmkasun) [reply en/docs/Learn/Tutorials/subscribe-to-an-api.md]: Thanks for pointing out this, I think the correct URL is

https://is.docs.wso2.com/en/5.9.0/learn/adding-and-configuring-a-service-provider/

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md (hunk 1 above)
Comment (tmkasun) [reply en/docs/Learn/Tutorials/subscribe-to-an-api.md]: Fixed in https://github.com/wso2/docs-apim/pull/326/commits/d3856d812d2db0cbb9b24c6374887d904f23ba61

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md (hunk 2 above)
Comment (tmkasun) [reply en/docs/Learn/Tutorials/subscribe-to-an-api.md]: fixed in https://github.com/wso2/docs-apim/pull/326/commits/d3856d812d2db0cbb9b24c6374887d904f23ba61

File: en/docs/Learn/Tutorials/subscribe-to-an-api.md (hunk 3 above)
Comment (tmkasun) [reply en/docs/Learn/Tutorials/subscribe-to-an-api.md]: fixed in https://github.com/wso2/docs-apim/pull/326/commits/d3856d812d2db0cbb9b24c6374887d904f23ba61

**Pull request merged**